src/wmnds/patterns/header-v2/_header-v2.njk

```diff
@@ -0,0 +1,163 @@
+        </a>
+      </div>
+      {#- If with a title #}
+      {% if params.withTitle -%}
+      <!-- Header title -->
+      <h1 class="wmnds-header__title wmnds-col-1 wmnds-col-sm-auto">
+        Reinstate your Direct Debit<br>(COVID-19)
+      </h1>
+      {%- elif params.megaMenu -%}
+      <nav id="{{params.id}}" class="wmnds-mega-menu">
+        <button class="wmnds-mega-menu__mobile-toggle wmnds-btn wmnds-btn--secondary wmnds-hide-desktop" aria-expanded="false" aria-controls="{{params.id}}-primary-menu">Menu
+          <svg class="wmnds-mega-menu__close-icon"><use xlink:href="#wmnds-general-cross" href="#wmnds-general-cross"></use></svg>
+        </button>
+        <div id="{{params.id}}-primary-menu" class="wmnds-mega-menu__scroll-controller">
+          <ul class="wmnds-mega-menu__primary-menu">
+          {% for navItem in navItems -%}
+            <li class="wmnds-mega-menu__primary-menu-item">
+              <a href="/{{ navItem.name | lower | replace(" ", "-") }}/" title="{{ navItem.name }} page" target="_self" class="wmnds-mega-menu__primary-menu-link" >
+                {{ navItem.name}}
+              </a>
+              {% if navItem.subnavItems -%}
+              <button class="wmnds-mega-menu__link-arrow-icon-btn" data-label="{{ navItem.name }}" aria-label="toggle {{ navItem.name }}" aria-controls="{{params.id}}-container"><svg class="wmnds-mega-menu__link-arrow-icon"><use xlink-href="#wmnds-general-chevron-right" href="#wmnds-general-chevron-right"></use></svg></button>
+              <div id="{{params.id}}-container-{{ loop.index }}" class="wmnds-mega-menu__container">
+                <div class="wmnds-container">
+                  <ul class="wmnds-mega-menu__sub-menu">
+                    {% for subnavItem in navItem.subnavItems -%}
+                    <li class="wmnds-mega-menu__sub-menu-item">
+                      <a href="/{{ navItem.name | lower | replace(" ", "-") }}/{{ subnavItem.name | lower | replace(" ", "-") }}/" class="wmnds-mega-menu__sub-menu-link">
+                        <svg class="wmnds-mega-menu__sub-menu-link-icon">
+                          <use xlink:href="#wmnds-general-star" href="#wmnds-general-star"></use>
+                        </svg> {{subnavItem.name}}</a>
+                        {%- if subnavItem.subnavItems %}
+                      <button class="wmnds-mega-menu__collapse-toggle" aria-expanded="false" aria-label="Toggle {{subnavItem.name}} menu" aria-controls="{{params.id}}-submenu-child-menu"><svg class="wmnds-mega-menu__link-arrow-icon"><use xlink-href="#wmnds-general-chevron-right" href="#wmnds-general-chevron-right"></use></svg></button>
+                      <ul id="{{params.id}}-submenu-child-menu-{{ subnavItem.name | lower | replace(" ", "-") }}-{{ loop.index }}" class="wmnds-mega-menu__sub-menu-child-menu">
+                        {% for subnavchildItem in subnavItem.subnavItems -%}
```

Comment:
Can you add some HTML comments throughout this looping section so in the code snippet it is easier to see what is going on and whether you are at level 1/2/3 etc.

Have a look at: https://deploy-preview-389--wmnetwork.netlify.app/components/disruption-indicators/ for some inspiration.